source/openapi-admin-v3.yaml

```diff
@@ -4387,143 +4387,6 @@ paths:
         "204":
           description: No Content
 
```

Comment:
Oh, I completely forgot we needed to do this in the Admin API docs. Thank you for noticing and removing the Edge Server endpoints.